cmd/nfd-topology-updater/main.go

```diff
@@ -86,6 +113,10 @@ func parseArgs(flags *flag.FlagSet, osArgs ...string) (*topology.Args, *resource
 		os.Exit(2)
 	}
 
+	if len(resourcemonitorArgs.KubeletConfigURI) == 0 {
+		resourcemonitorArgs.KubeletConfigURI = fmt.Sprintf("https://%s:%d/configz", nfdclient.NodeName(), kubeletSecurePort)
```

Comment:
Just wonder should we do any validation on `nfdclient.NodeName()`, e.g. error out if it's empty, WDYT?